November Newsletter

EdwinB12 opened the pull request.

EdwinB12 committed:
November Newsletter

davidwyld commented on _newsletters/2022-11-25-newsletter.md:
```diff
@@ -0,0 +1,136 @@
+
+* [LunchBytes Panel: Teaching Code](https://rse.shef.ac.uk/events/lunchbytes-2022-11-24.html) - As code and software become an ever bigger part of research (its likely that around 1/3 of researchers write code), coding and other digital skills become ever more important to researchers. Questions around teaching code were put to an online panel. The panel included [Emma Norling](https://www.sheffi ...
+
+#### News, Web and Publications
```
This heading is a bit repetitive of the one on line 17

EdwinB12 replied:
How do we feel about 'Web and Publications' instead? Is that still too repetitive?

davidwyld replied:
Follow your heart.

davidwyld commented on _newsletters/2022-11-25-newsletter.md:
```diff
@@ -0,0 +1,136 @@
+
+#### Research IT Training
+
+Research IT is providing a place for beginners or advanced users to expand their knowledge of HPC and different programming languages. The courses are part of the Doctoral Development Programme. For more information please visit our training registration web page (via VPN): [crs.shef.ac.uk](https://crs.shef.ac.uk).
```
I don't know that this list of sold out courses deserves a place above some scheduled courses with places available, maybe we should move it down.

EdwinB12 replied:
Yer thats fair, I'll move our two courses to the bottom of the training section

davidwyld reviewed (changes requested):
Nice one, there is A LOT in this newsletter! Couple of suggestions.

Robadob commented on _newsletters/2022-11-25-newsletter.md:
```diff
@@ -0,0 +1,136 @@
+
+* [Computing Insight UK 2022](https://www.scd.stfc.ac.uk/Pages/CIUK2022.aspx) - Manchester Central Convention Complex on Thursday 1 and Friday 2 December. The theme for the conference this year is "Sustainable HPC" with sub-themes including "Sustainable Skills", "Sustainable Funding" and "Net Zero Computing".
+* [PyData Global 2022](https://pdg22.wpengine.com) - 2022-12-01 to 2022-12-03, Online. The PyData Global conference is an opportunity to listen, learn, and share knowledge about best practices, new approaches, and emerging technologies for data management, processing, analytics, and visualization.
+* [University of Sheffield Inaugural Annual Open Research Lecture](https://www.eventbrite.co.uk/e/university-of-sheffield-inaugural-annual-open-research-lecture-tickets-440984976937) - Lecture given by Dr Tom Stafford titled 'Wide Open?'. Dr Tom Stafford will review concerns about Open Research practices: that they are unnecessary, performative, exclusive, neocolonial, a distracting example of da ...
```
When?

Robadob commented on _newsletters/2022-11-25-newsletter.md:
```diff
@@ -0,0 +1,136 @@
+* [University of Sheffield Inaugural Annual Open Research Lecture](https://www.eventbrite.co.uk/e/university-of-sheffield-inaugural-annual-open-research-lecture-tickets-440984976937) - Lecture given by Dr Tom Stafford titled 'Wide Open?'. Dr Tom Stafford will review concerns about Open Research practices: that they are unnecessary, performative, exclusive, neocolonial, a distracting example of da ...
+* [UK National Open Hackathon 2023](https://www.openhackathons.org/s/siteevent/a0C5e000005V6EDEA0/se000151) - Application Deadline January 16, 2023. Open Hackathons are multi-day intensive hands-on events designed to help computational scientists and researchers port and optimize their applications using GPUs.
+* [Free ARCHER2 course at Newcastle University covering Docker and Singularity](https://www.archer2.ac.uk/training/courses/221207-containers/) - This course is a run of the Docker and Singularity lessons in the Carpentries Incubator. 7-8th December.
+* [R course for Digital Humanities](https://n8cir.org.uk/events/r-digital-humanities/) - Following a similar format to the 2021 workshop, the course is a gentle introduction to R for text analysis. Over the course of two sessions you will be taught the basics of the powerful programming language before being provided with hands-on experience analysing long-form text in the RStudio development env ...
```
When?

Robadob commented on _newsletters/2022-11-25-newsletter.md:
```diff
@@ -0,0 +1,136 @@
+* [I-X Centre for AI in Science](https://ix.imperial.ac.uk/research-centre/aiscience/) -  Imperial’s I-X Centre for AI in Science. I-X is Imperial’s flagship AI initiative bringing AI together with interdisciplinarity and grand challenges. Our centre is dedicated to using AI to disrupt and advance Science, Engineering and Mathematics and is underpinned by core support from Schmidt Futures.
+
+
+### Events
```
Would be nice to have a consistent date format

* Thursday 1 and Friday 2 December
* 2022-12-01 to 2022-12-03
* January 16, 2023
* 27–31 March 2023

etc

Robadob replied:
Also they don;t appear to be in chronological order. Archer course is dec, but after something with a january deadline.

davidwyld replied:
> Would be nice to have a consistent date format

This should just be a running comment on newsletters at this point..

ns-rse replied:
ISO8601!

EdwinB12 replied:
Yup this is fair point, generally just copied from whatever format someone put in the Slack. Lazy Edwin! Anyway, I'll convert to ISO8601 and order chronologically

Robadob commented on _newsletters/2022-11-25-newsletter.md:
```diff
@@ -0,0 +1,136 @@
+
+This two half-day course is being run this month:
+
+* [28 November 2022 and 29 November 2022](https://rse.shef.ac.uk/training/workshop/2022-11-28-git-zero-hero)
```
Not worth advertising this course, it's already started.

Robadob replied:
Wasn't there a meeting last week to discuss new dates? (Not sure who to bother about this for more info, I just recall it being mentioned)

EdwinB12 replied:
We've decided some dates internally but not yet made the events. We decided it best not to advertise these yet until the eventbrite event has been created

Robadob replied:
Could atleast hint that ones for months x/y/z will be announced soon, so keep an eye out on the mailing list.

Robadob commented on _newsletters/2022-11-25-newsletter.md:
```diff
@@ -0,0 +1,136 @@
+* [Wellcome PhD studentships](https://pheds-dtc.ac.uk/). Training the next generation of researchers in high-quality research of complex multi-component public health interventions and policies.
+* [PhD at University of Bristol: Open spectroscopy in Environmental Sciences: Machine-learningaugmented analysis and interpretation ](https://www.bristol.ac.uk/media-library/sites/earthsciences/documents/phd-projects/2023-start/Byrne_UOB_2023_Studentships.pdf) PhD that is associated with the Bristol RSE group. The project title is “Open spectroscopy in Environmental Sciences: Machine-learning- au ...
+* [Get up to 12 days of software engineering support for your HPC/ML/GPU project](https://rse.shef.ac.uk/collaboration/tier2/). The Research Software Engineering team is offering to support researchers with the use of High Performance Computing (HPC), Machine Learning (ML) and Graphics Processors (GPUs) in their research projects. All researchers at our University can receive up to £5,000 of supp ...
+* [ARCHER2 Embedded CSE (eCSE) Support Call](https://www.archer2.ac.uk/ecse/calls/). The ARCHER2 eCSE call provides funding for up to 18 person months of an RSE/PDRA to work on codes to run on ARCHER2. The deadline for submitting documents for technical evaluations is 2022-10-04 16:00 BST with a final proposal deadline of 2022-10-25 16:00 BST.
```
The deadline for this funding call has passed

EdwinB12 replied:
Good spot, thanks

Robadob commented on _newsletters/2022-11-25-newsletter.md:
```diff
@@ -0,0 +1,136 @@
+* [Artificial intelligence hubs for real data and for scientific and engineering research](https://www.ukri.org/opportunity/artificial-intelligence-hubs-for-real-data-and-for-scientific-and-engineering-research/) -  Apply for funding for a research hub in either artificial intelligence (AI) for real data or AI for scientific and engineering research.
+* [Mathematical and computational foundations of artificial intelligence](https://www.ukri.org/opportunity/mathematical-and-computational-foundations-of-artificial-intelligence/). Apply for funding to advance the fundamental mathematical, statistical, and computational concepts that underpin artificial intelligence (AI).
+* [Wellcome PhD studentships](https://pheds-dtc.ac.uk/). Training the next generation of researchers in high-quality research of complex multi-component public health interventions and policies.
+* [PhD at University of Bristol: Open spectroscopy in Environmental Sciences: Machine-learningaugmented analysis and interpretation ](https://www.bristol.ac.uk/media-library/sites/earthsciences/documents/phd-projects/2023-start/Byrne_UOB_2023_Studentships.pdf) PhD that is associated with the Bristol RSE group. The project title is “Open spectroscopy in Environmental Sciences: Machine-learning- au ...
```
Deadline?

Robadob commented on _newsletters/2022-11-25-newsletter.md:
```diff
@@ -0,0 +1,136 @@
+
+* [Artificial intelligence hubs for real data and for scientific and engineering research](https://www.ukri.org/opportunity/artificial-intelligence-hubs-for-real-data-and-for-scientific-and-engineering-research/) -  Apply for funding for a research hub in either artificial intelligence (AI) for real data or AI for scientific and engineering research.
+* [Mathematical and computational foundations of artificial intelligence](https://www.ukri.org/opportunity/mathematical-and-computational-foundations-of-artificial-intelligence/). Apply for funding to advance the fundamental mathematical, statistical, and computational concepts that underpin artificial intelligence (AI).
+* [Wellcome PhD studentships](https://pheds-dtc.ac.uk/). Training the next generation of researchers in high-quality research of complex multi-component public health interventions and policies.
```
deadline?

Robadob replied:
Possibly a bit more context here too, expect phd studentships aren't of interest to most the readership (but what do I know someone submit it to the slack channel).

Robadob commented on _newsletters/2022-11-25-newsletter.md:
```diff
@@ -0,0 +1,136 @@
+#### Funding
+
+* [Artificial intelligence hubs for real data and for scientific and engineering research](https://www.ukri.org/opportunity/artificial-intelligence-hubs-for-real-data-and-for-scientific-and-engineering-research/) -  Apply for funding for a research hub in either artificial intelligence (AI) for real data or AI for scientific and engineering research.
+* [Mathematical and computational foundations of artificial intelligence](https://www.ukri.org/opportunity/mathematical-and-computational-foundations-of-artificial-intelligence/). Apply for funding to advance the fundamental mathematical, statistical, and computational concepts that underpin artificial intelligence (AI).
```
Deadline?

Robadob commented on _newsletters/2022-11-25-newsletter.md:
```diff
@@ -0,0 +1,136 @@
+
+### Opportunities
+
+#### Funding
```
Similar to events, consistent date format and chronological ordering would be nice, makes it alot easier to scan.

Robadob commented on _newsletters/2022-11-25-newsletter.md:
```diff
@@ -0,0 +1,136 @@
+* [ARCHER2 Embedded CSE (eCSE) Support Call](https://www.archer2.ac.uk/ecse/calls/). The ARCHER2 eCSE call provides funding for up to 18 person months of an RSE/PDRA to work on codes to run on ARCHER2. The deadline for submitting documents for technical evaluations is 2022-10-04 16:00 BST with a final proposal deadline of 2022-10-25 16:00 BST.
+* [Digital Infrastructure Incubator](https://www.codeforsociety.org/incubator/accepting-proposals) Deadline December 2, 2022. Code for Science & Society (CS&S) solicits expressions of interest from open source digital infrastructure teams to participate in the second cohort of the Digital Infrastructure Incubator (DII).
+* [CHIST-ERA Open & Re-usable Research Data & Software](https://www.chistera.eu/call-ord-announcement).  This call tackles the challenge of open research data and software from the perspective of their possible reuse. The objective is to create the conditions for research in any domain based on open or shared data and software. Call deadline: 2022-12-14 17:00 CET
+* [Oracle for Research Project Awards](https://www.oracle.com/research/project-awards/) Oracle Research (this years sponsors of the RSE conference) have an open grants program to sponsor research projects with free cloud credits to tackle computational challenges. The majority of cloud providers no longer offer free credits however Oracle grants include access to a variety of Oracle Cloud technol ...
```
deadline?

EdwinB12 replied:
Doesn't appear to be a deadline as far as I can tell. Worth putting something like 'Deadline N/A' or 'Open Call' or just leave blank?

Robadob replied:
Yeah one of those will suffice

EdwinB12 committed:
november newsletter changes from feedback

EdwinB12 committed:
Updated Date

EdwinB12 commented:
Thanks for the reviews @Robadob @davidwilby , believe I've addressed the changes: 

1. Changed all dates to 2022-11-29 format 
2. Ensure all events have location and date as first bit of information
3. Moved events and funding to chronological order
4. Removed events and funding opportunities that have passed
5. Removed Git training date and added a note to look out for newly advertised course dates
6. Moved git and deep learning training to the bottom of the training section

Robadob reviewed (approved):
Looks good to me.

There's still a couple of dates in the Web and Pubs section, but probably fine in that context.

EdwinB12 commented:
Thanks @Robadob , I'll wait for Davids review before merging. Happy with a merge commit or do you want it squashing?

Robadob commented:
> Thanks @Robadob , I'll wait for Davids review before merging. Happy with a merge commit or do you want it squashing?

Personally I'm a big fan of squashing then rebase + merge. As I like a linear history without junk split commits. But this isn't my repo, so not my problem.
